src/main/java/com/exactpro/th2/common/event/EventBuilder.java

```diff
@@ -70,33 +70,33 @@
     protected String description;
     protected Status status = Status.PASSED;
 
-    protected Event(Instant startTimestamp, @Nullable Instant endTimestamp) {
+    protected EventBuilder(Instant startTimestamp, @Nullable Instant endTimestamp) {
```

Comment:
constructors of this class should receive an instance of the EventFactory class to create sub events

Reply:
done